<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>alt::ComponentsWidget</class>
 <widget class="QWidget" name="alt::ComponentsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>429</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Components</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayoutLabels">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetMaximumSize</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutLabels">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetMaximumSize</enum>
       </property>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="componentsWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>10</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <widget class="QWidget" name="gridLayoutWidget_1">
           <layout class="QGridLayout" name="gridLayout_component">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>6</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="1" column="0">
             <widget class="QTreeView" name="componentsTreeView">
              <property name="contextMenuPolicy">
               <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
              </property>
              <property name="alternatingRowColors">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLineEdit" name="searchEdit">
              <property name="placeholderText">
               <string>Search...</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="gridLayoutWidget_2">
           <layout class="QGridLayout" name="gridLayout_description">
            <property name="leftMargin">
             <number>6</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>6</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="componentsLabel">
              <property name="text">
               <string>Description:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QStackedWidget" name="descriptionWidget">
              <widget class="QWidget" name="page_1">
               <layout class="QVBoxLayout" name="verticalLayout_1">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QTextBrowser" name="descriptionTextBrowser">
                  <property name="undoRedoEnabled">
                   <bool>false</bool>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                  <property name="textInteractionFlags">
                   <set>Qt::TextInteractionFlag::TextBrowserInteraction</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="page_2">
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QListView" name="listView">
                  <property name="viewMode">
                   <enum>QListView::ViewMode::ListMode</enum>
                  </property>
                  <property name="uniformItemSizes">
                   <bool>true</bool>
                  </property>
                  <property name="batchSize">
                   <number>100</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="gridLayoutWidget_3">
           <layout class="QGridLayout" name="gridLayout_packages">
            <property name="leftMargin">
             <number>6</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="1">
             <widget class="QLabel" name="packagesLabel">
              <property name="text">
               <string>Content:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QListView" name="contentListView">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="font">
               <font>
                <family>Monospace</family>
               </font>
              </property>
              <property name="contextMenuPolicy">
               <enum>Qt::ContextMenuPolicy::CustomContextMenu</enum>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
